### PR TITLE
translate 2014-11-28-ruby-2-2-0-preview2-released(zh_cn)

### PR DESCRIPTION
**Pull request by Airmacho**
screen shot 2015-05-31 at 00 25 44
Related #1074

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia @debbbbie @AndyHelix
@Zane5 @VirgilMing @boyishwei @dandananddada

Pleas help review this.

File: zh_cn/news/_posts/2014-11-28-ruby-2-2-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+layout: news_post
+title: "Ruby 2.2.0-preview2发布"
+author: "naruse"
+translator: "Will"
+date: 2014-11-28 09:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布，Ruby 2.2.0-preview2发布了。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-11-28-ruby-2-2-0-preview2-released.md]: **Ruby 2.2.0-preview2 发布了**

File: zh_cn/news/_posts/2014-11-28-ruby-2-2-0-preview2-released.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Ruby 2.2.0-preview2 是 Ruby 2.2.0 的第二个预览版，针对使用者日益增多的各类需求，进行了许多更新和修正，增加了许多新的功能。
+
+例如，符号 GC 使得符号对象可以被垃圾回收器回收了。这减少了符号的内存占用，因为在 2.2 版本之前，GC 都是不能回收符号对象的。因为 Rails 5.0 版本将支持对符号 GC，所以只支持 Ruby 2.2 或之后的版本（参考 [Rails blog post](http://weblog.rubyonrails.org/2014/8/20/Rails-4-2-beta1/)获取更多信息）。
+
+同时，新使用的增量垃圾回收器减少了停滞时间，对运行 Rails 应用非常有利。
+
+另一个与内存管理相关的功能是给`configure.in`增加了新的选项，可以启用 jemalloc，请参考 [Feature #9113](https://bugs.ruby-lang.org/issues/9113)。这个功能还在实验阶段，目前默认没有开启，我们需要收集实际使用和性能的数据来确认有良好的效果。确认之后这个功能会默认开启。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-11-28-ruby-2-2-0-preview2-released.md]: **给 `configure.in` 增加了**

Comment (Airmacho) [inline zh_cn/news/_posts/2014-11-28-ruby-2-2-0-preview2-released.md]: 没明白这里哪里错了？

Comment (hlcfan) [inline zh_cn/news/_posts/2014-11-28-ruby-2-2-0-preview2-released.md]: just missed spaces around English words.

Comment (Airmacho) [inline zh_cn/news/_posts/2014-11-28-ruby-2-2-0-preview2-released.md]: got it

**Comment by stomar** [comment]
Please squash into a **single commit** and force push your topic branch.

**Comment by JuanitoFatas** [comment]
Thanks @Airmacho! Squashed and merged manually as cb94668.

**Pull request closed without merging**